### ISSUE TITLE
Add an "attending" checkbox to the Event Attendance Page

### ISSUE THREAD

**Issue by adobrusky** [cseseniordesign/reservations#19]
As an admin, I want to be able to select an "attended" checkbox on the event attendance page, so that a member who attended can automatically have their tool permissions updated.
- [x] Add an "attending" checkbox on the event attendance page
- [x] Create technical and user documentation

**Comment by adobrusky** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Nebraska-Innovation-Studio-63111060cd14b400139f8047/issues/cseseniordesign/reservations/19?planning-poker) with Zenhub @taylor-runge @evanmielak @hadleysusie

**Comment by evanmielak** [comment]
Splitting this ticket into creating the UI checkbox component and actually creating the functionality for the checkbox because the ticket is blocked by another issue that will not be completed this sprint.
